python/Scripts/textures_patterns.json

```diff
@@ -0,0 +1,136 @@
+{
+  "texture_extensions": [
+    "exr",
```

Comment:
Would it make sense to include `hdr` ?

Reply:
It will be good for me if you check all JSON file data, because, as I mentioned above, it will be the default one.